cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/OrderedTableDefinition.java

```diff
@@ -55,7 +56,6 @@ public DatasetAdmin getAdmin(DatasetSpecification spec, ClassLoader classLoader)
   @Override
   public Table getDataset(DatasetSpecification spec, Map<String, String> arguments, ClassLoader classLoader)
     throws IOException {
-    OrderedTable table = tableDef.getDataset(spec.getSpecification(""), arguments, classLoader);
-    return new TableDataset(spec.getName(), table);
+    return tableDef.getDataset(spec.getSpecification(""), arguments, classLoader);
```

Comment:
doesn't this have to be new OrderedTable(spec.getName(), tableDef.getDataset(....))?